config/values/recruitmentcycle.yml

```diff
@@ -0,0 +1,2 @@
+recruitmentcycle:
```

Comment:
This is fine but not related to this PR?

Reply:
But can stay on this PR as only a config change that will be used in another PR in the near future